package.json

```diff
@@ -77,10 +75,12 @@
     "commander": "^9.1.0",
     "deep-object-diff": "^1.1.7",
     "eslint": "^8.21.0",
+    "fireworm": "^0.7.2",
```

Comment:
fixes `testem ci` error

Reply:
but what is it?

Reply:
the error?
or the package?

Reply:
I forgot what was the error...

Reply:
no this package. How does it fix n error?